src/commands/mod.rs

```diff
@@ -586,3 +586,54 @@ impl Formatter for PlaylistItems {
         )
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use tempfile::tempdir;
+    use std::io::Read;
+
+    #[test]
+    fn test_save_to_file_should_save_yaml_correctly() {
+        let dir = tempdir().unwrap();
+        let file_path = dir.path().join("test.yaml");
+
+        let manifest = EdgeAppManifest {
+            app_id: "test_app".to_string(),
+            settings: vec![Setting {
+                title: "username".to_string(),
+                type_: "string".to_string(),
+                default_value: "stranger".to_string(),
+                optional: true,
+                help_text: "An example of a setting that is used in index.html".to_string(),
+            }],
+            ..Default::default()
+        };
+
+        EdgeAppManifest::save_to_file(&manifest, &file_path).unwrap();
+
+        let mut file = File::open(&file_path).unwrap();
+        let mut contents = String::new();
+        file.read_to_string(&mut contents).unwrap();
```

Comment:
you can read to string in one liner.

let contents = fs::read_to_string(file_path)